docs/04-module-block-4/02-learning-goals.adoc

```diff
@@ -41,7 +41,7 @@ tbd.
   a software system.
 
 [[LG-4-3]]
-==== LG 4-3: Model Checking
+==== LG 4-3: Apply model checking
```

Comment:
As already written in my email, these three LGs (4-3 - 4-6) are too short (the titles not the content), please try to be more explicit.